public/javascripts/index.js

```diff
@@ -317,7 +318,7 @@ $(() => {
                             .split("\n")
                             .map(itemRef => {
                                 if (/^xb_room_number:/.test(itemRef)) {
-                                    roomNumber = itemRef.split(":")[1];
+                                    roomNumber = itemRef.split(":")[1];                                    
```

Comment:
This removes the whitespace at the end of the line.
```suggestion
                                    roomNumber = itemRef.split(":")[1];
```